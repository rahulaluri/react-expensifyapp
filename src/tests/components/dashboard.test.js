import ReactShallowRenderer from 'react-test-renderer/shallow';
import React from 'react';
import {shallow} from 'enzyme';
import ExpenseDashboard from '../../components/dashboard';

test('Should render the expense dashboard correctly', ()=>{
    const wrapper = shallow(<ExpenseDashboard />);
    expect(wrapper).toMatchSnapshot();
})
