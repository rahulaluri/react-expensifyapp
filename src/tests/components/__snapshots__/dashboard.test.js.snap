// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the expense dashboard correctly 1`] = `
<div>
  This is from my dashboard component!
  <Connect(ExpenseList) />
</div>
`;
